source/content/overview/index.md

```diff
@@ -0,0 +1,43 @@
+# Overview
+
+This section gives an overview on *why* and *how* the Brightway life cycle assessment (LCA) framework works, as well as some generic LCA methodology and implementation discussion. It is not designed for people learning about Brightway for the first time - please [see the Brightway book](https://learn.brightway.dev/) which is tailored for newcomers.
+
+```{note}
+If you have questions that the documentation cannot answer, you can [get in touch with the Brightway community or the Brightway developers](../contact/contact.md).
+```
+
+```{toctree}
+---
+hidden: true
+maxdepth: 2
+---
+self
+core
+inventory
+matrix
+io
+```
+
+## Brightway philosophy
+
+Brightway is designed for flexibility, and does not place many constraints on what you are allowed to do while using it. This can be dangerous, as we will happily allow you to delete your database or enter numerical values which can break calculations. If you are first getting started, it's safest to follow the examples and cheat sheet.
+
+## Brightway core concepts introduction
+
+**Projects**: A Brightway project is an isolated set of data sufficient to do LCA calculations. In the core Brightway libraries, each project is implemented as a separate subdirectory with a SQLite3 database and directory for datapackages.
```

Comment:
These should be defined in [the glossary](https://docs.brightway.dev/en/latest/content/other/glossary.html), which I added a while ago. This is core Sphinx functionality which I find very useful. Then, these terms can always be cross-referenced.

Reply:
I agree with you - maybe we can put the glossary as part of the overview then? I think people will more naturally start here, but need to know these ideas before moving forward with the rest of this content.

Reply:
Yes, I think so. The page can be moved anywhere. It just has to contain a `(glossary)=` header.

Reply:
Started in b491e3aaa32b3af049a9bf5a2955e16ed2061c68